.cirrus.yml

```diff
@@ -23,7 +23,7 @@ env:
     # GCE project where images live
     IMAGE_PROJECT: "libpod-218412"
     # VM Image built in containers/automation_images
-    IMAGE_SUFFIX: "c20241010t105554z-f40f39d13"
+    IMAGE_SUFFIX: "c20241107t210000z-f41f40d13"
```

Comment:
Line 20 needs to `s/39/40/`. Is there a simple-quick way to do this other than git pulls and branch magic?